keyword_spotting_data_generator/evaluation/evaluate.py

```diff
@@ -100,13 +101,18 @@ def main():
 
             extracted_audio_times = extractor.extract_keywords(data)
 
-            # TODO :: count how many window has been extracted and compare it against true count
+            true_count = line[-1]
+            if true_count == len(extracted_audio_times):
+                success += 1
+            else:
+                cp.print_info("extracted ", len(extracted_audio_times), " windows, but count is ", true_count)
 
             # TODO :: might be good idea to update threshold if the accuracy is way too low
 
     cp.print_progress("evaluation is completed for ", keyword, " - ", total)
 
-    # TODO :: calculate accuracy and report metrics
+    accuracy = (success / total) * 100
+    cp.print_info("extractor achieved ", accuracy, "% accuracy")
```

Comment:
let's use `cp.print_progress` and I think it's a good idea to apply round with 2 decimal